OpenDigger_Community_Handbook.md

```diff
@@ -0,0 +1,205 @@
+# Values
+
+Help open source projects with open-source data analysis.
+
+# Roles and Responsibilities
+
+## Committer
+
+* Facilitate the creation and advancement of software
+* Answer questions about the project
+* Report on weekly community progress in the regular meeting
+* Develop a roadmap
+* Respond to issues on the repository
+* Review and merge pull requests
+* Regularly check the repository for stale content
+* monitor issue tracker and pull requests
+* steer the architecture of the software
+## Contributor
+
+* Attend community meetings
+* Post ideas or problems for new features as issues
+* Respond to  issues on the repository
+* Review pull requests
+## Meeting Host
+
+* Send a reminder about community meetings
+* Prepare weekly community meeting
+* Host weekly community meeting
+* Make sure meeting minutes are kept
+* Collect themes from a week in the OpenDigger community
+* The host can be changed without any punishment before 4 pm every Thursday
+* The host can be changed by deducting the credits that the hostshould getone hour before the meeting
+* Count credits of participants in the weekly meeting
+    * Format: **yaml**
```

Comment:
Maybe we can add [YAML](https://yaml.org/) link here.

Reply:
OK